pyproject.toml

```diff
@@ -32,11 +32,11 @@ requests = "^2.32.0"
 cohere = { version = "^5.5.4", optional = true }
 anthropic = { version = "^0.29.0", optional = true }
 transformers = { version = "^4.39.3", optional = true, extras=["torch"] }
-huggingface-hub = { version = ">=0.13", optional = true }
+huggingface-hub = { version = "^0.24.0", optional = true }
 boto3 = { version = "^1.34.119", optional = true }
 snowflake-sqlalchemy = { version = "^1.6.1", optional = true }
 pinecone-client = { version = "^3", optional = true }
-pymongo = { version = "*", optional = true }
+pymongo = { version = "4.8.0", optional = true }
```

Comment:
Should we add a caret for this one as well?

Reply:
Yep, fixed.